logging/efk-logging-deploy.adoc

```diff
@@ -1,6 +1,6 @@
 :context: efk-logging-deploy
 [id='efk-logging-deploy_{context}']
```

Comment:
Because we're not nesting subassemblies, you don't really need the context statement here.